pkg/apis/webconsole/validation/validation.go

```diff
@@ -50,14 +51,27 @@ func ValidateWebConsoleConfiguration(config *v1.WebConsoleConfiguration, fldPath
 		validationResults.AddWarnings(field.Invalid(fldPath.Child("metricsPublicURL"), "", "required to view cluster metrics in the console"))
 	}
 
-	for i, scriptURL := range config.ExtensionScripts {
-		if _, scriptURLErrs := ValidateSecureURL(scriptURL, fldPath.Child("extensionScripts").Index(i)); len(scriptURLErrs) > 0 {
+	if len(config.LogoutPublicURL) > 0 {
+		_, urlErrs := ValidateURL(config.LogoutPublicURL, fldPath.Child("logoutPublicURL"))
```

Comment:
why isn't this required to be secure?

Reply:
Does it need to be? As I understand it, it is just a redirect URL after logout.

https://github.com/openshift/origin-web-console/blob/ea328a53a741c7c460914106fb0274f378195183/app/scripts/controllers/util/logout.js#L25-L26

FWIW, it wasn't required to be secure before.

https://github.com/openshift/origin-web-console-server/blob/master/pkg/apis/webconsole/validation/validation.go#L17

Reply:
> Does it need to be?

I like all things secure.  If it wasn't before, I guess it doesn't have to be today.